src/compile/data/bin.ts

```diff
@@ -15,7 +15,7 @@ export namespace bin {
     return `format('${format}', ${expr})`;
   }
 
-  function parse(model: Model): Dict<VgTransform[]> {
+  export function parseUnit(model: Model): Dict<VgTransform[]> {
```

Comment:
The reason for the change is that `parseUnit` was a public const which was set to the value of `parse`. `parse` was a closure variable, so its types were not considered public, resulting in declaration errors. Instead of simply changing parse to `parseUnit` (which seemed right, since they were equivalent), I can revert it to `parse` and write 
```ts
export const parse: (model: Model) => Dict<VgTransform[]> = parseUnit;
```
if that is preferred. I simply thought this would be less verbose.